crc32.go

```diff
@@ -4,22 +4,17 @@ const (
 	crc32Polynomial = uint32(0xffffffff)
 )
 
-// computeCRC32 computes a CRC32
-// https://stackoverflow.com/questions/35034042/how-to-calculate-crc32-in-psi-si-packet
+// this solution based on vlc implementation + static crc table (1kb additional memory on start, without reallocations)
```

Comment:
Could you put this comment above `updateCRC32` instead ?

And could you replace it with : 

```
// Based on VLC implementation using a static CRC table (1kb additional memory on start, without 
// reallocations): https://github.com/videolan/vlc/blob/master/modules/mux/mpeg/ps.c
```

Reply:
Done